backend/apps/llm/services.py

```
@@ -94,28 +107,92 @@ def _set_environment_variables(self):
         except Exception as e:
             logger.error(f"[{self.request_id}] 设置环境变量失败: {e}", exc_info=True)
 
-    def review_code(self, code_context, mr_info=None, repo_path=None, commit_range=None, custom_prompt=None):
+    def review_code(self, code_context=None, mr_info=None, repo_path=None,
+                    commit_range=None, custom_prompt=None):
         """
-        Review code using Claude CLI (新版本)
+        代码审查入口 - 根据 provider 分发到不同的审查策略
+
+        双模式接口：
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)